tests/smoke/test_respondent_home.py

```diff
@@ -29,3 +33,4 @@ def test_can_access_required_services(self):
 
         # Then
         self.assertEqual(resp['ready'], True, resp)
+        self.assertEqual(resp['name'], "respondent-home-ui", resp)
```

Comment:
Has this been proven to work in each part of the pipeline? (Including where Frontstage is also deployed to the same domain).

Reply:
I tried running this in latest and pre-prod with the [pipeline PR](https://github.com/ONSdigital/ras-deploy/pull/86). The test passes, but it's because we're using the internal [Cloud Foundry deployed app](https://github.com/ONSdigital/ras-deploy/pull/86/files#diff-390068eef6009ad2e88f5fbd837803dcL192) instead of the shared domain URL for the smoke test. This was a workaround for the /info end point test on a shared domain. Not sure how reliable using a separate URL is for smoke tests, though.